aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -171,30 +189,121 @@ T quantize_val(double scale, int64_t zero_point, float value) {
   int64_t qvalue;
   constexpr int64_t qmin = std::numeric_limits<typename T::underlying>::min();
   constexpr int64_t qmax = std::numeric_limits<typename T::underlying>::max();
-  checkZeroPoint<typename T::underlying>("quantize_val", zero_point);
```

Comment:
why is this removed?

Reply:
We were calling this for each value, moved it out of the for loop.
Saves about 20ms on mobile :)